<?xml version="1.0"?>
<launch>

  <node pkg="robotour_waypoint_corrector" name="wp_corrector" type="wp_corrector" output="screen">
   <param name="map_frame" value="/odom"/> 
   <param name="robot_frame" value="/base_link"/>
   <param name="map_topic" value="/move_base/local_costmap/costmap"/>
   <param name="path_min_width" value="1.0"/>
   <param name="path_max_width" value="10.0"/>
   <param name="obstacle_bloat" value="1.0"/>
  </node>

</launch>
